python/sparkdl/estimators/keras_image_file_estimator.py

```diff
@@ -131,41 +133,6 @@ def setParams(self, inputCol=None, inputImageNodeName=None, outputCol=None,
         kwargs = self._input_kwargs
         return self._set(**kwargs)
 
-    def fit(self, dataset, params=None):
-        """
-        Fits a model to the input dataset with optional parameters.
-
-        .. warning:: This returns the byte serialized HDF5 file for each model to the driver.
-                     If the model file is large, the driver might go out-of-memory.
-                     As we cannot assume the existence of a sufficiently large (and writable)
-                     file system, users are advised to not train too many models in a single
-                     Spark job.
-
-        :param dataset: input dataset, which is an instance of :py:class:`pyspark.sql.DataFrame`.
-                        The column `inputCol` should be of type `sparkdl.image.imageIO.imgSchema`.
-        :param params: An optional param map that overrides embedded params. If a list/tuple of
-                       param maps is given, this calls fit on each param map and returns a list of
-                       models.
-        :return: fitted model(s). If params includes a list of param maps, the order of these
-                  models matches the order of the param maps.
-        """
-        self._validateParams()
```

Comment:
Currently this is unused - let's call this in `fitMultiple` to validate our params before running any Spark jobs?

Side note: it's a little weird that `_validateParams` only checks the input & output column parameters, it'd be nice to also validate that the other params are specified.

Reply:
The original intention is to just check if `inputCol` and `outputCol` are set. Other `Param`s do not seem necessary to be checked, eg. `HasKerasModel` and other `HasKeras*` parameters seem to be used differently than the usual `Param`s.

Should we make a jira issue to check these, since that requires considerable work and not captured here? Please advise! Thanks! :)